brainiak/isfc.py

```diff
@@ -145,6 +164,12 @@ def isfc(D, collapse_subj=True, return_p=False,
         A random number generator instance to define the state of the
         random permutations generator.
 
+    float_type : either float16, float32, or float64
+        depending on the required precision
+        and available memory in the system.
+        cast all the arrays generated during the execution to
+        specified float type, in order to save memory

```

Comment:
Same comment about writing regular sentences.

Reply:
Thank you, fixed accordingly.